test/uk/gov/hmrc/helptosaveproxy/connectors/DWPConnectorSpec.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosaveproxy.connectors
+
+import org.scalamock.scalatest.MockFactory
+import org.scalatest.EitherValues
+import play.api.Configuration
+import play.api.libs.json.Json
+import uk.gov.hmrc.helptosaveproxy.TestSupport
+import uk.gov.hmrc.helptosaveproxy.config.WSHttpProxy
+import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
+import uk.gov.hmrc.http.logging.Authorization
+import uk.gov.hmrc.helptosaveproxy.config.AppConfig.dwpUCClaimantCheckUrl
+import uk.gov.hmrc.helptosaveproxy.testutil.UCClaimantTestSupport
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, ExecutionContext, Future}
+
+class DWPConnectorSpec extends TestSupport with MockFactory with EitherValues with UCClaimantTestSupport {
+
+  lazy val mockHTTPProxy = mock[WSHttpProxy]
+
+  def testDWPConnectorImpl = new DWPConnectorImpl(
+    fakeApplication.configuration ++ Configuration()) {
```

Comment:
you can add the below line to the TestSupport and just use it here like : `new DWPConnectorImpl (configuration)`

`  lazy implicit val configuration: Configuration = fakeApplication.injector.instanceOf[Configuration]`

Reply:
ok maybe you could help me with this tomorrow?

Reply:
ok thanks, I've managed to do that now :)